Libraries/DeprecatedPropTypes/DeprecatedTextStylePropTypes.js

```diff
@@ -74,6 +74,17 @@ const DeprecatedTextStylePropTypes = {
       | 'proportional-nums',
     >,
   >),
+  /** [TODO(OSS Candidate ISS#2710739)
```

Comment:
Where is this deprecated file still used?

Reply:
I actually don't know.   But it contains all the same props as the other files like ViewConfig so I made it parallel.